python/util/templates.py

```diff
@@ -1,8 +1,11 @@
+import json
+import flask
```

Comment:
the stuff in this package (`sdk`) should not generally import flask.

we have to be general-purpose here. flask is a specific artifact of our html pushing mechanism. it only belongs in frontend.

Reply:
We use flask in 4 other files inside sdk

Reply:
I only see 3, prior to this patch. 1 of them is OK, it is just using a helper. the other two are bad and should be fixed.